include/volk/volk_version.h.in

```diff
@@ -0,0 +1,50 @@
+/* -*- C -*- */
+/*
+ * Copyright 2020 Free Software Foundation, Inc.
+ *
+ * This file is part of GNU Radio
+ *
+ * GNU Radio is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation; either version 3, or (at your option)
+ * any later version.
+ *
+ * GNU Radio is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with GNU Radio; see the file COPYING.  If not, write to
+ * the Free Software Foundation, Inc., 51 Franklin Street,
+ * Boston, MA 02110-1301, USA.
+ */
+
+#ifndef INCLUDED_VOLK_VERSION_H
+#define INCLUDED_VOLK_VERSION_H
+
+#include <volk/volk_common.h>
+
+__VOLK_DECL_BEGIN
+
+/*
+ * define macros for the Volk version, which can then be used by any
+ * project that #include's this header, e.g., to determine whether
+ * some specific API is present and functional.
+ */
+
+#define VOLK_VERSION_MAJOR @VOLK_VERSION_MAJOR@
+#define VOLK_VERSION_MINOR @VOLK_VERSION_MINOR@
+#define VOLK_VERSION_MAINT @VOLK_VERSION_MAINT@
+
+/*
+ * VOLK_VERSION % 100 is the MAINT level
+ * (VOLK_VERSION / 100) % 100 is the MINOR version
+ * (VOLK_VERSION / 100) / 100 is the MAJOR version
+ */
```

Comment:
The comment says VOLK 2.1 translates into `002001000` but the actual implementation is `020100`. So this should be fixed one way or the other.

Reply:
hmm ... no, I think both are consistent with each other. A `VOLK_VERSION ` of `020100`, according to the comment, will result in `MAINT == 00` (`VOLK_VERSION % 100`), `MINOR == 01` (`(VOLK_VERSION / 100) % 100`), and `MAJOR == 02` (`(VOLK_VERSION / 100) / 100`). Note the `% 100` and `/ 100`, not `1000` -- which would result in the triplets you note.

Reply:
That said, I'll fix `level` -> `version`

Reply:
I read that wrong. Thanks for the clarification. The `level` -> `version` change would be good.

Reply:
Changed